magefiles/utils.go

```diff
@@ -17,6 +17,10 @@ func binaryWithExt(name string) string {
 	return name
 }
 
+func checkOs() string {
```

Comment:
Hmmm. I would want to see this merged into one function with https://github.com/armadaproject/armada/blob/042d12ea31a20da8bdda9eba94372fb57d41d2bb/magefiles/utils.go#L50C1-L53C2.

Or, at the very least, use the same naming scheme for the function

Reply:
Would this be good to go?

```go
func onWindows() bool {
	return runtime.GOOS == "windows"
}
```